test/test.pl

```diff
@@ -154,6 +210,37 @@ sub _cmd
     }
     return ($? >> 8, join('',@out));
 }
+sub _cmd3
+{
+    my ($cmd) = @_;
+
+    my $tmp = "$$opts{tmp}/tmp";
+    my $pid = fork();
+    if ( !$pid )
+    {
+        exec('bash', '-o','pipefail','-c', "($cmd) 2>$tmp.e >$tmp.o");
+    }
+    waitpid($pid,0);
```

Comment:
This could just be replaced by
```
system('bash', '-o','pipefail','-c', "($cmd) 2>$tmp.e >$tmp.o");
```

Compared with `_cmd()`, the `$ENV{TEST_PRECMD}` functionality has been lost (although we don't use it that much these days, anyway).

Reply:
OK